agents/executor.py

```
@@ -2,34 +2,35 @@
 from protocols.loader import load_protocol
 from utils.logger import log
 
-def execute_task(protocol_name='default_protocol'):
+
+def execute_task(protocol_name="default_protocol"):
     """Execute a specific protocol and return the outcome"""
     # Try database tracker first, fall back to file tracker
     try:
         from utils.db_tracker import track_outcome
     except:
```

Flake8 (7.2.0)

[error] do not use bare 'except'

(E722)